docs/design/sheets/axis-id-selection.md

```
@@ -0,0 +1,227 @@
+type UserPresence = {
+  selection?: SelectionPresence;
+  activeTabId?: string;
+} & User;
+```
+
+### Coordinate Conversion Layer
+
+Axis ID ↔ visual coordinate conversion happens **only at the Store boundary**.
+The Sheet engine keeps using `Ref` internally.
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)